great_expectations/datasource/data_connector/configured_asset_sql_data_connector.py

```diff
@@ -62,20 +54,19 @@ def __init__(
         id: Optional[str] = None,
     ) -> None:
         """
-        InferredAssetDataConnector for connecting to data on a SQL database
 
         Args:
             name (str): The name of this DataConnector
             datasource_name (str): The name of the Datasource that contains it
             execution_engine (ExecutionEngine): An ExecutionEngine
-            data_asset_name_prefix (str): An optional prefix to prepend to inferred data_asset_names
-            data_asset_name_suffix (str): An optional suffix to append to inferred data_asset_names
```

Comment:
Removing since it is an `inferred`-level paramter

Reply:
@billdirks The data_asset_name_prefix and data_asset_name_suffix are part of the `ConfiguredAssetSqlDataConnector` functionality as well.  Here is an example:
```
    "data_connectors": {
        "my_test_postgresql_db_configured_connector_0": {
            "module_name": "great_expectations.datasource.data_connector",
            "class_name": "ConfiguredAssetSqlDataConnector",
            "assets": {
                "new_titanic_copy_0": {
                    "module_name": "great_expectations.datasource.data_connector.asset",
                    "class_name": "Asset",
                    "schema_name": "new_schema",
                    "table_name": "new_titanic",
                    "include_schema_name": True,
                    "data_asset_name_prefix": "test__",
                    "data_asset_name_suffix": "__asset",
                    "splitter_method": "split_on_column_value",
                    "splitter_kwargs": {
                        "column_name": "Sex",
                    },
                },
            },
        },
        "my_test_postgresql_db_configured_connector_1": {
            "module_name": "great_expectations.datasource.data_connector",
            "class_name": "ConfiguredAssetSqlDataConnector",
            "assets": {
                "titanic": {
                    "module_name": "great_expectations.datasource.data_connector.asset",
                    "class_name": "Asset",
                    "schema_name": "my_schema",
                    "include_schema_name": True,
                    "data_asset_name_prefix": "test__",
                    "data_asset_name_suffix": "__asset",
                    "splitter_method": "split_on_column_value",
                    "splitter_kwargs": {
                        "column_name": "Age",
                    },
                    "sampling_kwargs": {
                        "p": 5.0e-1,
                    },
                },
                "titanic_copy_1": {
                    "module_name": "great_expectations.datasource.data_connector.asset",
                    "class_name": "Asset",
                    "schema_name": "my_schema",
                    "table_name": "titanic",
                    "include_schema_name": True,
                    "data_asset_name_prefix": "test__",
                    "data_asset_name_suffix": "__asset",
                    "splitter_method": "split_on_column_value",
                    "splitter_kwargs": {
                        "column_name": "Age",
                    },
                    "sampling_method": "_sample_using_random",
                    "sampling_kwargs": {
                        "p": 5.0e-1,
                    },
                },
            },
        },
```